packages/eslint-plugin/src/rules/contextual-lifecycle.ts

```diff
@@ -31,7 +31,8 @@ export default createESLintRule<Options, MessageIds>({
     },
     schema: [],
     messages: {
-      contextuaLifecycle: `This lifecycle method is not called for {{decorator}}`,
+      contextualLifecycle:
+        'Lifecycle out of context for "@{{classDecoratorName}}()"',
```

Comment:
Yes, we can change this. I went with this message because we're using it in `contextual-decorator` which exists for the same purpose, except that is for decorators, not lifecycle methods. That being said, should we change `contextual-decorator` to better explicit like this one?

Reply:
I haven't looked at it, not super high priority to get everything changed in this same PR though, we can always tackle others you haven't already touched in follow ups if needed